scripts/build_loop.dev.sh

```diff
@@ -1,21 +1,86 @@
 #!/bin/bash
 
-if [ "$1" != "" ]; then
-    branch="$1"
-    echo "Get latest from $branch"
+SCRIPT="$(basename "${0}")"
+SCRIPT_DIRECTORY="$(dirname "${0}")"
+ROOT_DIRECTORY="${SCRIPT_DIRECTORY}/../"
+
+error() {
+  echo "ERROR: ${*}" >&2
+  echo "Usage: ${SCRIPT} [-s|--save-current <build_tag>] [-f|--fetch-new <build_tag>]" >&2
+  echo "Parameters:" >&2
+  echo " -s|--save-current <build_tag>     folder name of build to save e.g. `build_277`" >&2
+  echo " -f|--fetch-new <build_tag>      tag name of build to fetch e.g. `build_281`" >&2
+  exit 1
+}
+
+info() {
+  echo "INFO: ${*}" >&2
+}
+
+buildLoop(){
+  LoopWorkspace/Scripts/workspace-prepare.sh Tidepool
+
+  ## LoopOverride.xcconfig with no `CRITICAL_ALERTS_ENABLED`
+  sed -i '' 's/CRITICAL_ALERTS_ENABLED//g' LoopWorkspace/Tidepool/Loop/LoopOverride.xcconfig
+
+  LoopWorkspace/Scripts/workspace-build.sh -o "$(pwd)" Tidepool
+
+  cp -r "$(pwd)"/Build/Products/*  "$(pwd)"
+}
+
+if [ ${#} -lt 1 ]; then
+  error "Missing arguments"
+fi
+
+SAVE_BUILD=
+FETCH_BUILD=
+BUILD_ONLY=NO
+while [[ $# -gt 0 ]]
+do
+  arg="$1"
+
+  case $arg in
+    -s|--save-current)
+      SAVE_BUILD="${2}"
+      shift 2
+      echo SAVE_BUILD = $SAVE_BUILD
+      ;;
+    -f|--fetch-new)
+      FETCH_BUILD="${2}"
+      shift 2
+      echo FETCH_BUILD = $FETCH_BUILD
+      ;;
+    -b|--build-only)
```

Comment:
[nit] If this is still an optional, include it in the usage description above? Or is this `BUILD_ONLY` really just the same as neither `-s` nor `-f` specified?